src/components/Input.tsx

```diff
@@ -86,7 +86,8 @@ class Input extends React.PureComponent<InputProps, InputState> {
       () => {
         Animated.timing(this.state.placeholderMarginTop, {
           toValue: 0,
-          duration: 200
+          duration: 200,
+          useNativeDriver: false
```

Comment:
shouldn't we prefer using native driver?

Reply:
https://reactnative.dev/blog/2017/02/14/using-native-driver-for-animated#caveats